packages/components/skeleton/src/skeleton.vue

```diff
@@ -28,34 +28,15 @@
 <script lang="ts">
 import { defineComponent, computed } from 'vue'
 import { useThrottleRender } from '@element-plus/hooks'
-import SkeletonItem from './item.vue'
+import SkeletonItem from './skeleton-item.vue'
+import { skeletonProps } from './skeleton'
 
 export default defineComponent({
   name: 'ElSkeleton',
   components: {
     [SkeletonItem.name]: SkeletonItem,
```

Comment:
There's a little problem. 
It should be 
```ts
    SkeletonItem,
```

Otherwise, the Volar can't detect type of component